packages/cms/src/components/Viz/Options.jsx

```diff
@@ -103,7 +103,7 @@ class Options extends Component {
       let background;
       if (backgroundColor) background = getBackground(node);
       saveElement(node,
-        {filename: filename(title), type},
+        {filename: `${filename(title)}.${type}`},
```

Comment:
I believe this may have unintended consequences. The `saveElement` function expects 2 keys, "filename" and "type", and without specifying type this may cause them all to use the fallback type ("PNG"), and would then name the files like this `my-title.png.png`.

Reply:
I'm unsure of the history of this, but your reasoning is solid so I've pushed a reversion. @perpetualgrimace could you chime in on whether this change was made to cover some bug you discovered?